tests/baselines/reference/tscWatch/programUpdates/updates-moduleResolution-when-resolveJsonModule-changes.js

```diff
@@ -117,14 +117,6 @@ Shape signatures in builder refreshed for::
 /user/username/projects/myproject/a.ts (computed .d.ts)
 
 PolledWatches::
-/user/username/projects/myproject/data.json:
-  {"pollingInterval":500} *new*
-/user/username/projects/myproject/node_modules/@types:
-  {"pollingInterval":500} *new*
-/user/username/projects/node_modules/@types:
-  {"pollingInterval":500} *new*
-

```

Comment:
These use to be deleted and created again when module resolution settings changed